workspace/assets/css/lib/class-transform.less

```diff
@@ -72,20 +74,11 @@
 	.scale-y-2@{suffix}                    {#transform(~'scaleY(2)');}
 }
 
-.translate-left-none-on-hover {
-	#hover({
-		#transform(~'translateX(0)');
-	});
-}
-
-.translate-y-none-on-hover {
-	#hover({
-		#transform(~'translateY(0)');
-	});
-}
+//HOVER
+//TODO DEPRECATE
+.translate-left-none-on-hover      {#hover({#transform(~'translateX(0)');});}
+.translate-y-none-on-hover         {#hover({#transform(~'translateY(0)');});}
 
-.translate-y-minus-full-on-hover {
-	#hover({
-		#transform(~'translateY(-100%)');
-	});
-}
+.translate-x-0-on-hover            {#hover({#transform(~'translateX(0)');});}
+.translate-y-0-on-hover            {#hover({#transform(~'translateY(0)');});}
+.translate-y-minus-full-on-hover   {#hover({#transform(~'translateY(-100%)');});}
```

Comment:
The commit message does not mention this change. Is it ok ?

Reply:
Yep. New Line 84 replace line 87-89, line 82 and 83 are for replacing the old line 75-85 but still present has line 79-80 in the commit. So nothing really new exception translate-{x,y}-0-on-hover.